sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Reverted to flattening `KeyVaultProperties` in `ServiceAccountEncryptionProperties` as service side does not support passing empty object for this property.
```

Comment:
`Reverted to flattening` is a bit misleading as if the result is to use flattening, `Reverted the falttening of` may be better?

Reply:
Good catch, fixed.